src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -1,9 +1,48 @@
+<!-- 
+    Custom element for showing a table of transactions.
+    Can be configured to show:
+     - The transaction history of a wallet (set attribute `type` to "txlist" and `wallet` to the wallet's alias)
+     - The UTXO list of a wallet (set attribute `type` to "utxo" and `wallet` to the wallet's alias)
+     - The transaction history of all the user's wallets (set attribute `type` to "txlist" and `wallet` to null)
+     - The UTXO list of all the user's wallets (set attribute `type` to "utxo" and `wallet` to null)
+    The uses Specter's API to fetch transactions every time the user changes its parameters.
+    These parameters are configured using components included in this table element:
+     - Tabs for switching between History (`type="txlist"`) and UTXO (`type="utxo"`) list types
+     - Sorting transactions by any column (ascending or descending)
+     - Search bar to filter transactions using free text search
+     - Configuring limit for transactions displayed per page
+     - Pagination, jumping between pages and showing the total pages count
+     - Export - export the full (unfiltered) list from which the table is drawing its data as CSV (optionally with price history)
+    
+    The table also supports:
+     - Showing amounts in either BTC or sats (set attribute `btc-unit` to either "btc" or "sat")
+     - Showing prices next to the amounts (set attribute `price` to the BTC price and symbol to the symbol of the currency you're pricing at)
+     - Showing validated blockhash column (set attribute `blockhash` to either "true" or "false")
+     - Limiting txs/utxos to just those associated with the specified `data-service-id`.
+     - Hide columns by hide-columns="category time"
+     - Enable scrollbar
+     - Toggle for scrollbar
+
+- Usage -
+
+<tx-table
+    btc-unit="{{ specter.unit }}"
+    hide-sensitive-info="{{ specter.hide_sensitive_info | lower }}"
+    type="utxo" 
+    hide-columns="category time txid confirmations action" 
+    blockhash="true" 
+    hide-switcher="true"
+    wallet="{{ wallet.alias }}"
+    scrollbar-height="333px"
+    scrollbar-default="off"
+    compact-switch="true">
+</tx-table> 
+-->
```

Comment:
`#}`  end jinja2 comment.

Reply:
Added in https://github.com/cryptoadvance/specter-desktop/pull/1580/commits/05c5a62e0bac83eab31dfda5d2d935b3bb67adef